docs/src/test/scala/docs/http/scaladsl/SprayJsonExampleSpec.scala

```diff
@@ -75,17 +83,22 @@ class SprayJsonExampleSpec extends WordSpec with Matchers {
       implicit val orderFormat = jsonFormat1(Order)
 
       // (fake) async database query api
-      def fetchItem(itemId: Long): Future[Option[Item]] = ???
-      def saveOrder(order: Order): Future[Done] = ???
+      def fetchItem(itemId: Long): Future[Option[Item]] = Future {
+        orders.filter(o => o.id == itemId) match {
```

Comment:
FYI this is a `.find` operation basically, it’s better than filter because it stops iterating once 1 element is found; Filter will continue going through the entire list

Reply:
Thank you very much for your review @ktoso 👍 , and `Find` is surely better than `Filter`. I will change it!